pmlc/dialect/pxa/transforms/vectorize.h

```diff
@@ -13,11 +13,12 @@ static constexpr const char *kVectorizeStrategy_Simple = "simple";
 // Explicitly attempt to vectorize on a specific loop/index/size
 mlir::LogicalResult performVectorization(mlir::AffineParallelOp op,
                                          mlir::BlockArgument index,
-                                         unsigned vectorSize);
+                                         unsigned vectorSize, bool loopMathOp);
 
 // Attempt to vectorize a loop on the stride 1 index of its output
 mlir::LogicalResult vectorizeOverOutputs(mlir::AffineParallelOp op,
-                                         unsigned vecSize);
+                                         unsigned vecSize,
+                                         bool loopMathOp = false);
```

Comment:
is this ` = false` needed? Maybe we should keep the same code style as `vecSize`.

Reply:
Because func `vectorizeOverOutputs` is called by other pass, like `fusion` and `subgroup`, if we do not set a default value here, we should explicitly add this augument when calling in above passes.

Reply:
I see, thanks.